### ISSUE TITLE
Double Icon for AsciidocFX in Ubuntu's Dock

### ISSUE THREAD

**Issue by Autiwa** [asciidocfx/AsciidocFX#403]
I struggled for months with double icons on Ubuntu for AsciidocFX. 

For context, you define a favorite app by creating a .desktop icon in ~/.local/share/applications (either automatically or by hand). 

I've just realized that the *StartupWMClass* in the .desktop file is wrong for AsciidocFX. 

The wrong line was:
`StartupWMClass=com.kodebu.boot.AppStarter`

You need this line:
`StartupWMClass=com.kodedu.boot.AppStarter`

In short, there's a typo somewhere with kodebu instead of kodedu. I don't know why, I don't know where it is. I downloaded the tar.gz in there, that's why I'm assuming the typo is in the repo rather than in Ubuntu. 

To summarize, I don't have an issue anymore, I'm just leaving this here in case it helps someone. 

source for the solution I've found: https://github.com/AppImage/appimaged/issues/36#issuecomment-292076548

**Comment by rahmanusta** [comment]
Thanks for reporting, I will look into this.

**Comment by rahmanusta** [comment]
Hi, I searched kodebu and couldn't find in anywhere in source code. I wonder how this is possible.

**Comment by Autiwa** [comment]
I tried before to do this exact same thing in the .tar.gz with grep, with no results. 

I don't really know the mechanism behind .desktop files so I'm not entirely sure my findings mean there's something going on in asciidocFX in particular and not in Ubuntu in itself. 

If you agree, I propose we close this ticket for now, and leave it as is for other people struggling with the same problem.

**Comment by rahmanusta** [comment]
Ok, thanks @Autiwa
